x-pack/plugins/triggers_actions_ui/public/application/hooks/use_load_rules.ts

```diff
@@ -1,193 +0,0 @@
-/*
```

Comment:
Just wanted to double-check that no one else is using these methods

Reply:
issue created https://github.com/elastic/kibana/issues/148610